ingestion_script/ingest_folder_source.py

```
@@ -214,7 +216,10 @@ def _ingest_folder_source(
     #     )
     #     add_summary_in_chunks_func = add_summary_in_chunks
     db_service.create_schema(db_table_schema)
+    LOGGER.info(f"Found {len(files_info)} files to ingest")
     try:
+        if len(files_info) == 0:
+            raise Exception("No files found to ingest")
```

Can we raise an error and not exception ? like valueError